res/text/bms_page_es-419.txt

```diff
@@ -0,0 +1,9 @@
+Modo
```

Comment:
I am not sure in my opinion (because i don't know Spanish), but this line should explicitly mean that it's mode of _playing_ (it's a title for choice normal/no-margin-for-mistake/no-possibility-to-fail/automatic). There is a possibility that more "mode" choices will exist in that UI area in future.